### PR TITLE
Expose noteKeys on createjs function

### PR DESCRIPTION
**Pull request by noelmugnier**
I've exposed the notesKeys (ntKeys) param to createjs function because i didn't figure out how to generate a js with notes without it :)

#### Checklist

- [x] only relevant code is changed (make a diff before you submit the PR)
- [x] run tests `npm run test`
- [x] tests are included
- [x] documentation is changed or added

### CODE REVIEW

**Comment by adrai** [comment]
Is there any particular reason why you're using the createJs function and not the xliff2js function https://github.com/locize/xliff/blob/master/test/test.js#L211 ?

File: test/test.js [hunk 1]
```diff
@@ -100,6 +100,22 @@ describe('single', () => {
     )
   })
 
+  test("createjs", (fn) => (done) => {
```
Comment (adrai) [inline test/test.js:103]: can you also add a createXliff test?

**Comment by adrai** [comment]
rest lgtm

**Comment by noelmugnier** [comment]
> Is there any particular reason why you're using the createJs function and not the xliff2js function https://github.com/locize/xliff/blob/master/test/test.js#L211 ?

Mainly because i don't start from xliff file, i'm building a svelte components parser which extract id|content|note from html tags. Then depending of user format i use createxliff12 or createjs to get a .xlf or .json file

File: test/test.js (hunk 1 above)
Comment (noelmugnier) [reply test/test.js:103]: sure, createxliff and createxliff12 added

**Comment by adrai** [comment]
Thank you for your contribution.
It's included in v5.7.0

**Pull request merged**